resources/dashboard.rb

```diff
@@ -37,6 +38,7 @@
   grafana_options = {
     host: new_resource.host,
     port: new_resource.port,
+    url_path_prefix: new_resource.url_path_prefix,
```

Comment:
I think it would be better to only pass in if it is set, and to not default it to an empty value
```suggestion
    url_path_prefix: new_resource.url_path_prefix if new_resource.url_path_prefix
```

Reply:
@Xorima could you then suggest how you would like to see api_helper library be modified ?
for now i tried to make it dead simple with less code as possible

Reply:
Do you need some additional help with this still?

Reply:
we pass `nil` now instead of empty string. but `url_path_prefix` option needs to be passed to the library because it is so easy to implement a new resource without the support for this feature.
this is the main intention.
am i confusing anything ? do we understand each other ?

Reply:
Ah yes, I see that now. You can disregard this.